### DISCUSSION TITLE
can it be used for multi-regression?

### DISCUSSION THREAD

**Discussion by yqh-1037** [pyg-team/pytorch-frame#465] [Q&A]
In Class Dataset(ABC), I found this 'target_col (str, optional): The column used as target.
 (default: :obj:`None`)'maybe it can only be used for single-regression? 
       thanks a lot!

**Comment by akihironitta** [comment]
Hi @yqh-1037, I believe we don't have any datasets for such regression tasks, but I'm sure it'd be a great addition to PyTorch Frame! Would you be interested in contributing?
